source/base/NexusApp.h

```diff
@@ -55,6 +55,8 @@ namespace nexus {
     G4String trkact_name_; ///< Name of the chosen tracking action
     G4String stkact_name_; ///< Name of the chosen stacking action
 
+    G4bool pman_; ///< True if the persistency manager is set
+
```

Comment:
I don't know what's the convention in nexus, but usually in C++ you could simply check if `pm_` is a `nullptr`.

I like that there is a more explicit variable, though, so for me this is fine.

Reply:
If you think this way it is easier to understand what the code does, then I would keep it.